include/exiv2/photoshop.hpp

```diff
@@ -47,16 +47,16 @@ struct EXIV2API Photoshop {
   /// @return 0 if successful;<BR>
   ///   3 if no data for psTag was found in pPsData;<BR>
   ///  -2 if the pPsData buffer does not contain valid data.
-  static int locateIrb(const byte* pPsData, size_t sizePsData, uint16_t psTag, const byte** record,
```

Comment:
Two comments here:
1) Why to change the signature only in the header and not in the .cpp too?
2) After looking at the code, I think it would even make more sense to pass the 2 last parameters by reference instead of by pointer. Then we do not need to check if those variables are `nullptr` or not.

Reply:
Done.